pkg/env/integersetting.go

```diff
@@ -34,7 +39,7 @@ func (s *IntegerSetting) IntegerSetting() int {
 		return s.defaultValue
 	}
 	v, err := strconv.Atoi(val)
-	if err != nil {
+	if err != nil || (v < s.minimumValue) || (v > s.maximumValue) {
 		return s.defaultValue
```

Comment:
Should we log a warning here?

Reply:
I would love to, but I cannot use logger in this pkg (cyclic import).